test/integration/skipports/skip_ports_test.go

```diff
@@ -0,0 +1,85 @@
+package skipports
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/linkerd/linkerd2/testutil"
+)
+
+var TestHelper *testutil.TestHelper
+
+var (
+	skipPortsNs               = "skip-ports-test"
+	booksappDeployments       = []string{"books", "traffic", "authors", "webapp"}
+	httpResponseTotalMetricRE = regexp.MustCompile(
+		`route_response_total\{direction="outbound",dst="books\.skip-ports-test\.svc\.cluster\.local:7002",classification="failure".*`,
+	)
+)
+
+func TestMain(m *testing.M) {
+	TestHelper = testutil.NewTestHelper()
+	os.Exit(m.Run())
+}
+
+//////////////////////
+/// TEST EXECUTION ///
+//////////////////////
+
+func TestSkipInboundPorts(t *testing.T) {
+	ctx := context.Background()
+
+	err := TestHelper.CreateDataPlaneNamespaceIfNotExists(ctx, skipPortsNs, nil)
```

Comment:
Can I suggest to use `TestHelper.WithDataPlaneNamespace()` instead? As we keep adding integration tests, when they are run sequentially (like in the ARM environment), the cluster sometimes runs out of resources. `WithDataPlaneNamespace()` will clean up resources when the test is done.